components/infobox/commons/infobox.lua

```diff
@@ -243,8 +243,10 @@ end
 
 --- Returns completed infobox
 function Infobox:build()
-    if self.chronologyContent ~= nil then
-        self.content:node(self.chronologyContent)
+    if not Table.isEmpty(self.chronologyContent) then
+        for _, chronologyRow in ipairs(self.chronologyContent) do
```

Comment:
With this for loop, you shouldn't need the isEmpty check

Reply:
true, my bad^^